package.json

```diff
@@ -9,40 +9,39 @@
     "calculation",
     "calc"
   ],
-  "main": "dist/index.js",
+  "main": "src/index.js",
   "types": "types/index.d.ts",
   "files": [
-    "dist",
+    "src",
     "types",
     "LICENSE"
   ],
   "scripts": {
     "prepare": "pnpm run build && tsc",
-    "build": "rimraf dist && babel src --out-dir dist --ignore src/__tests__/**/*.js && jison src/parser.jison -o dist/parser.js",
+    "build": "jison src/parser.jison -o src/parser.js",
     "lint": "eslint src && tsc",
     "pretest": "pnpm run build",
-    "test": "uvu -r @babel/register src/__tests__"
+    "test": "uvu src/__tests__"
   },
   "author": "Andy Jansson",
   "license": "MIT",
   "repository": "https://github.com/postcss/postcss-calc.git",
   "eslintConfig": {
     "extends": [
-      "eslint:recommended",
-      "plugin:import/recommended"
+      "eslint:recommended"
     ],
+    "env": {
+      "node": true,
+      "es2017": true
+    },
+    "ignorePatterns": ["src/parser.js"],
```

Comment:
This is generated code, it will never pass linting. Before it used to be output in `dist/` so it was not checked.